lib/mayaUsd/ufe/Utils.cpp

```diff
@@ -625,17 +634,26 @@ Ufe::Attribute::Type usdTypeToUfe(const PXR_NS::SdrShaderPropertyConstPtr& shade
 PXR_NS::SdfValueTypeName ufeTypeToUsd(const std::string& ufeType)
 {
     // Map the USD type into UFE type.
-    static const std::unordered_map<Ufe::Attribute::Type, PXR_NS::SdfValueTypeName> sUfeTypeToUsd {
-        { Ufe::Attribute::kBool, PXR_NS::SdfValueTypeNames->Bool },           // bool
-        { Ufe::Attribute::kInt, PXR_NS::SdfValueTypeNames->Int },             // int32_t
-        { Ufe::Attribute::kFloat, PXR_NS::SdfValueTypeNames->Float },         // float
-        { Ufe::Attribute::kDouble, PXR_NS::SdfValueTypeNames->Double },       // double
-        { Ufe::Attribute::kString, PXR_NS::SdfValueTypeNames->String },       // std::string
-        { Ufe::Attribute::kEnumString, PXR_NS::SdfValueTypeNames->Token },    // TfToken
-        { Ufe::Attribute::kInt3, PXR_NS::SdfValueTypeNames->Int3 },           // GfVec3i
-        { Ufe::Attribute::kFloat3, PXR_NS::SdfValueTypeNames->Float3 },       // GfVec3f
-        { Ufe::Attribute::kDouble3, PXR_NS::SdfValueTypeNames->Double3 },     // GfVec3d
-        { Ufe::Attribute::kColorFloat3, PXR_NS::SdfValueTypeNames->Color3f }, // GfVec3f
+    static const std::unordered_map<Ufe::Attribute::Type, PXR_NS::SdfValueTypeName> sUfeTypeToUsd
+    {
+        { Ufe::Attribute::kBool, PXR_NS::SdfValueTypeNames->Bool },               // bool
+            { Ufe::Attribute::kInt, PXR_NS::SdfValueTypeNames->Int },             // int32_t
+            { Ufe::Attribute::kFloat, PXR_NS::SdfValueTypeNames->Float },         // float
+            { Ufe::Attribute::kDouble, PXR_NS::SdfValueTypeNames->Double },       // double
+            { Ufe::Attribute::kString, PXR_NS::SdfValueTypeNames->String },       // std::string
+            { Ufe::Attribute::kEnumString, PXR_NS::SdfValueTypeNames->Token },    // TfToken
+            { Ufe::Attribute::kInt3, PXR_NS::SdfValueTypeNames->Int3 },           // GfVec3i
+            { Ufe::Attribute::kFloat3, PXR_NS::SdfValueTypeNames->Float3 },       // GfVec3f
+            { Ufe::Attribute::kDouble3, PXR_NS::SdfValueTypeNames->Double3 },     // GfVec3d
+            { Ufe::Attribute::kColorFloat3, PXR_NS::SdfValueTypeNames->Color3f }, // GfVec3f
+#if (UFE_PREVIEW_VERSION_NUM >= 4015)
+            { Ufe::Attribute::kFilename, PXR_NS::SdfValueTypeNames->Asset },
```

Comment:
Should you have comments equivalent to those above that show the data type? ie. // GfVec3d

Reply:
Are these comments adding value in this function?

Reply:
Not sure, but I would think for consistency they should either be removed or added.

Reply:
Also, isn't this a trivial change? If not, by all means ignore this.